engine/names/extract/build.rs

```diff
@@ -1,8 +1,16 @@
-use hax_frontend_exporter::{DefId, DefPathItem, DisambiguatedDefPathItem};
 use serde_json;
 use serde_json::Value;
 use std::process::{Command, Stdio};
 
+/// Instead of depending on `hax_frontend_exporter` (that links to
+/// rustc and exposes a huge number of type definitions and their
+/// impls), we just inline a small module here that contains the three
+/// type definition we need. See the module for complementary
+/// informations.
+#[path = "../../../frontend/exporter/src/types/def_id.rs"]
```

Comment:
I'm not actually sure if this is publishable.

Reply:
Ah, that's a legitimate question!
But this one is not supposed to be published, it's solely for the ocaml build